public/app/app.ts

```diff
@@ -96,8 +96,9 @@ export class GrafanaApp {
       setLocationSrv(locationService);
       setTimeZoneResolver(() => config.bootData.user.timezone);
       // Important that extensions are initialized before store
```

Comment:
Nit: Can we update the comment? It is a bit confusing to see the `initExtensions` below the `configureStore`

Reply:
updated with https://github.com/grafana/grafana/pull/41850/commits/7d12658e5ec3425ed24fdd5aaac4823afb60e7a5